micro-ui/web/micro-ui-internals/packages/modules/hrms/src/pages/EmployeeDetails.js

```
@@ -62,6 +63,9 @@ const Details = () => {
     return <Loader />;
   }
 
+  console.log(":data of the employee", data);
+  console.log("role data is", data?.Employees?.[0]?.user.roles.map((ele) => t(`ACCESSCONTROL_ROLES_ROLES_` + ele?.code)));
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)